plugins/modules/nfs_exports_info.py

```diff
@@ -0,0 +1,133 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright: (c) 2025, Samaneh Yousefnezhad <s-yousefenzhad@um.ac.ir>
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: nfs_exports_info
+
+short_description: Extract folders, IPs, and options from /etc/exports
```

Comment:
```suggestion
short_description: Extract folders, IPs, and options from C(/etc/exports)

version_added: 10.7.0
```

Reply:
There's no `version_added` entry yet. Right now 11.0.0 is the right number, but that will change beginning of next week to 11.1.0.